examples/common/imgui_ui/windows/light.cpp

```diff
@@ -0,0 +1,164 @@
+/*
+ *
+ *    Copyright (c) 2023 Project CHIP Authors
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+#include "light.h"
+
+#include <imgui.h>
+
+#include <math.h>
+
+#define SKIP_ENDPOINT_CONFIG_INCLUDE
```

Comment:
This should not be needed after #25128 (assuming that passes CI and can be merged)